lib/libutee/default_bold.c

```diff
@@ -0,0 +1,1009 @@
+/*
```

Comment:
+1 to commit. You can also ask Joakim to be the referee. Either way is fine for me.